### PR TITLE
Test with cloudbuild

### PR DESCRIPTION
**Pull request by erinpentecost**

### CODE REVIEW

**Review by mattayes — APPROVED** [review]
:shipit:

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    - get
+    - -t
+    - -v 
+    - ./...
+
+- id: 'test'
+  name: 'gcr.io/cloud-builders/go'
+  entrypoint: '/bin/sh'
+  env:
+    - CGO_ENABLED=1
```
Comment (vitaminmoo) [inline cloudbuild.yaml:15]: Q: Is this actually needed?

Comment (vitaminmoo) [reply cloudbuild.yaml:15]: Context is that we have it disabled for the main repo so if it's required there is some sort of issue

Comment (erinpentecost) [reply cloudbuild.yaml:15]: Yeah it's used for the race condition checker

**Review by vitaminmoo — APPROVED** [review]

**Pull request merged**